examples/showcase/room-scale/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Room Scale</title>
+    <meta name="description" content="Room Scale – A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+    <script src="js/collider.js"></script>
```

Comment:
I've been using `components/` as a folder name so you know where to grab the reusable components.